mappings/net/minecraft/client/gl/GlExtensionHandler.mapping

```diff
@@ -59,59 +59,138 @@ CLASS none/cbg net/minecraft/client/gl/GlExtensionHandler
 	FIELD z GL_SOURCE0_RGB I
 	METHOD a init ()V
 	METHOD a deleteShader (I)V
+		ARG 0 shader
 	METHOD a multiTexCoord2f (IFF)V
+		ARG 0 target
+		ARG 1 s
+		ARG 2 t
 	METHOD a getProgrami (II)I
+		ARG 0 program
+		ARG 1 pname
```

Comment:
I still don't agree with having these 1:1 with GL argument names but it should at _least_ be `pName`.